AISKU/src/Initialization.ts

```diff
@@ -463,10 +463,14 @@ export class Initialization implements IApplicationInsights {
 
                 // Back up the current session to local storage
                 // This lets us close expired sessions after the cookies themselves expire
-                const ext = appInsightsInstance.appInsights.core['_extensions'][PropertiesPluginIdentifier];
-                if (ext && ext.context && ext.context._sessionManager) {
-                    ext.context._sessionManager.backup();
-                }
+                arrForEach(appInsightsInstance.appInsights.core['_extensions'], (ext: any) => {
```

Comment:
Does this change add extra loop compared with previous code? Looks like previously you only need to check if "ext" exists, with current change, you'll loop the extensions array and check every ext's identifier. 🤔

Reply:
The previous code actually didn't work, because it "assumed" that the array contained a key with the string value of PropertiesPluginIdentifier... Which is doesn't.

So the loop is required to find the properties plugin within the array -- this part is actually a bug fix that was found by adding additional type checks caused by the refactoring.